.github/workflows/pr.yml

```diff
@@ -0,0 +1,14 @@
+name: PR
```

Comment:
this is the main PR trigger - just calls the other build steps now as no deploy steps were actually ran on PRs